src/app/components/vc-inspection/ParsedCredentialInfo.tsx

```diff
@@ -26,53 +26,31 @@ function HLineWithText({ text }: { text: string }) {
  * Dates validity, listed data for issuer and subject, errors, proofs, parsed JSON
  */
 export default function ParsedCredentialInfo({ inspectedResult, className, ...props }: ParsedCredentialInfoProps) {
-  const validSchema = inspectedResult.type === 'ValidCredential';
-  const dates = validSchema ? inspectedResult.calculatedAttributes.validityDates : undefined;
-  const standard = new StandardRetriever(Standards.W3C_V2);
-  const standardDates = dates ? standard.extractOk(dates) : undefined;
+  // TODO: More dynamic types
+  const standard = new StandardRetriever(
+    inspectedResult.parsedJson.type === 'CBOR' ? Standards.MDOC : Standards.W3C_V2,
+  );
 
-  let subjectValues = undefined;
-  let issuerValues = undefined;
-  if (validSchema) {
-    const subject = inspectedResult.parsedJson.credentialSubject;
-    subjectValues = fromJSON(subject);
-    const issuer = inspectedResult.parsedJson.issuer;
-    if (typeof issuer == 'string') {
-      issuerValues = [labeledValue('id', node(issuer))];
-    } else if (issuer) {
-      issuerValues = fromJSON(issuer);
-    }
-  } else {
-    const json = inspectedResult.parsedJson.payload;
-    if (isStrRecord(json)) {
-      if ('credentialSubject' in json && isStrRecord(json.credentialSubject)) {
-        subjectValues = fromJSON(json.credentialSubject);
-      }
-      if ('issuer' in json) {
-        if (isStrRecord(json.issuer)) {
-          issuerValues = fromJSON(json.issuer);
-        } else if (typeof json.issuer === 'string') {
-          issuerValues = [labeledValue('id', node(json.issuer))];
+  const dates = standard.extractOk(inspectedResult.calculatedAttributes.validityDates);
+
+  const subject = standard.extractOk(inspectedResult.calculatedAttributes.credentialSubject);
```

Comment:
We should have some way to show if issuer or credential subject is not present. Given an error or similar. How are zods errors now handled?

Reply:
Issuer has been made on this